tests/performance_tests/README.md

```diff
@@ -0,0 +1,605 @@
+* `target_tps`                  Target transactions per second to generate/send.
+* `tps_limit_per_generator`     Maximum amount of transactions per second a single generator can have.
+* `log_dir`                     set the logs directory
+</details>
+
+### Transaction Generator
+`./build/tests/trx_generator/trx_generator` can be configured using the following command line arguments:
+
+<details>
+    <summary>Expand Argument List</summary>
+
+* `--chain-id arg`                  set the chain id
+* `--handler-account arg`           Account name of the handler account for
+                                    the transfer actions
+* `--accounts arg`                  comma-separated list of accounts that 
+                                    will be used for transfers. Minimum 
+                                    required accounts: 2.
+* `--priv-keys arg`                 comma-separated list of private keys in
+                                    same order of accounts list that will 
+                                    be used to sign transactions. Minimum 
+                                    required: 2.
+* `--trx-expiration arg` (=3600)    transaction expiration time in seconds.
+                                    Defaults to 3,600. Maximum allowed: 
+                                    3,600
+* `--trx-gen-duration arg` (=60)    Transaction generation duration 
+                                    (seconds). Defaults to 60 seconds.
+* `--target-tps arg` (=1)           Target transactions per second to 
+                                    generate/send. Defaults to 1 
+                                    transaction per second.
+* `--last-irreversible-block-id arg`      Current last-irreversible-block-id (LIB
+                                    ID) to use for transactions.
+* `--monitor-spinup-time-us arg` (=1000000)
+                                    Number of microseconds to wait before 
+                                    monitoring TPS. Defaults to 1000000 
+                                    (1s).
+* `--monitor-max-lag-percent arg` (=5)    Max percentage off from expected 
+                                    transactions sent before being in 
+                                    violation. Defaults to 5.
+* `--monitor-max-lag-duration-us arg` (=1000000)
+                                    Max microseconds that transaction 
+                                    generation can be in violation before 
+                                    quitting. Defaults to 1000000 (1s).
+* `--log-dir arg`                   set the logs directory
+</details>
+
+## Result Reports
+
+### Performance Test
+
+Command used to run test and generate report:
+
+``` bash
+.build/tests/performance_tests/performance_test.py --test-iteration-duration-sec 10 --final-iterations-duration-sec 30 --save-json True
+```
+
+`InitialMaxTpsAchieved` - the max TPS throughput achieved during initial, short duration test scenarios to narrow search window
```

Comment:
A sentence describing where these will be found is appropriate.

Reply:
Addressed: [b641304](https://github.com/AntelopeIO/leap/pull/352/commits/b641304b5c62607aaac45e49a9d1de9cf701bba6)